addons/payment_xendit/static/src/js/payment_form.js

```diff
@@ -138,7 +138,17 @@ paymentForm.include({
      */
     _xenditHandleResponse(err, token, processingValues) {
         if (err) {
-            this._displayErrorDialog(_t("Payment processing failed"), err.message);
+            let err_message = err.message;
+
+            // If the error is API_VALIDATION_ERROR: there's an invalid input
+            if (err.error_code === 'API_VALIDATION_ERROR') {
+                let err_obj = err.errors[0]
+                // Issue is with string format for input
+                if (err_obj.type === 'string.regex.base') {
```

Comment:
Do you need to always restrict to `string.regex.base`  or we can always show the first error if there is `err.errors`? 
 ```suggestion
                if (err_obj.type === 'string.regex.base') {  // Wrong field format.
```

Reply:
right, I tried around with wrong form inputs and as long as it's 'API_VALIDATION_ERROR', the message will always be there. Will remove the if condition